sdk/translation/Azure.AI.Translation.Document/tests/samples/SampleSnippets.cs

```diff
@@ -71,21 +77,33 @@ public void BadRequestSnippet()
         }
 
         [Test]
-        [Ignore("Samples not working yet")]
+        [SyncOnly]
         public void DocumentTranslationInput()
         {
             #region Snippet:DocumentTranslationSingleInput
+#if SNIPPET
             Uri sourceSasUri = new Uri("<source SAS URI>");
             Uri frenchTargetSasUri = new Uri("<french target SAS URI>");
-
+#else
+            Uri sourceSasUri = CreateSourceContainer(oneTestDocuments);
+            Uri frenchTargetSasUri = CreateSourceContainer(oneTestDocuments);
+#endif
             var input = new DocumentTranslationInput(sourceSasUri, frenchTargetSasUri, "fr");
             #endregion
 
             #region Snippet:DocumentTranslationMultipleInputs
+
+#if SNIPPET
             Uri arabicTargetSasUri = new Uri("<arabic target SAS URI>");
             Uri spanishTargetSasUri = new Uri("<spanish target SAS URI>");
             Uri source1SasUri = new Uri("<source1 SAS URI>");
             Uri source2SasUri = new Uri("<source2 SAS URI>");
+#else
+            Uri source1SasUri = CreateSourceContainer(oneTestDocuments);
```

Comment:
see my comment above about how this is not needed for this sample

Reply:
Yea it makes sense that we don't actually need to create containers, but it seems like there is some form of format checking that takes place. A `System.UriFormatException : Invalid URI` exception is thrown if the sample input is used. I thought that creating an actual container would suffice, but you're right that it is unnecessary.

I could however just use any other sample uri, like the url for the github repo for example. Or is there a standard one that I should use?

Reply:
I think once you remove the inheritance from `DocumentTranslationLiveTestBase` back to `SamplesBase<DocumentTranslationTestEnvironment>` you should be ok

Reply:
Unfortunately the format exception is still thrown. I believe it is thrown by the Uri constructor as it does its own checks on the input string format.